stdlib/src/collections/dict.mojo

```diff
@@ -894,8 +894,12 @@ struct Dict[K: KeyElement, V: CollectionElement](
     fn _insert(inout self, owned key: K, owned value: V):
         self._insert(DictEntry[K, V](key^, value^))
 
-    fn _insert(inout self, owned entry: DictEntry[K, V]):
-        self._maybe_resize()
+    fn _insert[
+        safe_context: Bool = False
+    ](inout self, owned entry: DictEntry[K, V]):
+        @parameter
+        if safe_context == False:
```

Comment:
```suggestion
        if not safe_context:
```

Reply:
I just made this change internally to the synced PR and I'll land this soon.  Thanks! 👍